exchange/bitswap/bitswap_test.go

```diff
@@ -35,30 +34,6 @@ func TestClose(t *testing.T) {
 	bitswap.Exchange.GetBlock(context.Background(), block.Key())
 }
 
-func TestProviderForKeyButNetworkCannotFind(t *testing.T) { // TODO revisit this
```

Comment:
what you describe is not what is being tested here. all it appears to be testing is that a context expires